introduction.md

```diff
@@ -1,3 +1,9 @@
 I am Thaichor.
 
-My favorte Code School path is Ruby.
\ No newline at end of file
+My favorte Code School path is Ruby.
+
+Skill
+===
+* Git
+* Javascript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?